src/concrete/ml/sklearn/tree_to_numpy.py

```diff
@@ -333,6 +351,129 @@ def tree_to_numpy(
     # but also rounding the threshold such that they are now integers
     q_y = tree_values_preprocessing(onnx_model, framework, output_n_bits)
 
-    _tree_inference, onnx_model = get_equivalent_numpy_forward_from_onnx(onnx_model)
+    _tree_inference, onnx_model = get_equivalent_numpy_forward_from_onnx(
+        onnx_model, check_model=True, lsbs_to_remove=lsbs_to_remove
```

Comment:
`check_model` is True by default !

Reply:
mypy will drive me crazy!

I had an error where he confused the typing of lsb_to_remove and the one of check_onnx, that's why I specified the check_model argument to avoid warnings from mypy.

Reply:
this is weird, even `lsbs_to_remove=lsbs_to_remove` ? This would be a big mypy flaw, could you maybe quickly try again and copy paste the error message it gives ? I'm curious !